src/main/java/io/github/shafthq/shaft/gui/element/ElementActionsHelper.java

```diff
@@ -102,7 +102,13 @@ private static boolean isValidToCheckForVisibility(By elementLocator, boolean ch
     }
 
     private static boolean isSafariBrowser() {
-        return DriverFactoryHelper.getTargetBrowserName().toLowerCase().contains("safari");
+        boolean isSafariBrowser= false;
```

Comment:
Why is it throwing a null? Is the user deleting the target browser property manually or what?

Reply:
Let's try to prevent the error from the source rather than try/catching it. I think it may be related to the new properties management system, this issue is a regression as it wasn't there before, so let's investigate the root cause some more to ensure that no other sleeper issues arise.

Reply:
Fixed and pushed 
the problem was when calling getExpectedExceptions() and calling isSafariBrowser() when getting getTargetBrowserName() it was null as it wasn't initialized here

Reply:
I just added a browserstack thread to the teating pipeline. Is this only happening with browserstack mobile?